rsvps/_rsvp.php

```diff
@@ -33,18 +34,19 @@ function get_rsvps(array $args = []): array
 /**
  * Retrieves the rsvps for a given event
  *
- * @param int   $event_id
- * @param array $args additional WP_Query args to add or overload
+ * @param int|int[] $event_id
+ * @param array     $args additional WP_Query args to add or overload
  *
  * @return WP_Post[]|int[]
  */
-function get_event_rsvps(int $event_id, array $args = []): array
+function get_event_rsvps($event_id, array $args = []): array
 {
-    $meta_query = $args['meta_query'] ?: [];
+    $meta_query = $args['meta_query'] ?? [];
```

Comment:
TIL about the "Null Coalescing operator" ... I would have thought this was a typo, but knowing you, I researched first. Interesting!
https://www.tutorialspoint.com/what-does-double-question-mark-operator-mean-in-php